eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -280,22 +283,17 @@ stages:
       - ${{ each artifact in parameters.Artifacts }}:
         - ${{if ne(artifact.skipPublishDevFeed, 'true')}}:
           - pwsh: |
-              Get-ChildItem $(Pipeline.Workspace)/${{parameters.ArtifactName}}
-              New-Item -Type Directory -Name ${{artifact.safeName}} -Path $(Pipeline.Workspace)
-              $underscorePrefix = "${{artifact.name}}"
-              $dashPrefix = "${{artifact.name}}".Replace("_", "-")
-              Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}/$dashPrefix-[0-9]*.[0-9]*.[0-9]*a[0-9]* $(Pipeline.Workspace)/${{artifact.safeName}}
-              Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}/$underscorePrefix-[0-9]*.[0-9]*.[0-9]*a[0-9]* $(Pipeline.Workspace)/${{artifact.safeName}}
-              Get-ChildItem $(Pipeline.Workspace)/${{artifact.safeName}}
-
-              $fileCount = (Get-ChildItem $(Pipeline.Workspace)/${{artifact.safeName}} | Measure-Object).Count
+              $packageDirectory = "${{artifact.name}}".Replace("_", "-")
+              echo "##vso[task.setvariable variable=Package.Name]$packageDirectory"
+          - pwsh: |
+              $fileCount = (Get-ChildItem $(Pipeline.Workspace)/${{parameters.ArtifactName}}/$packageDirectory  | Measure-Object).Count
               if ($fileCount -eq 0) {
                 Write-Host "No alpha packages for ${{artifact.safeName}} to publish."
                 exit 0
               }
 
-              twine upload --repository $(DevFeedName) --config-file $(PYPIRC_PATH) $(Pipeline.Workspace)/${{artifact.safeName}}/*a*.whl
+              twine upload --repository $(DevFeedName) --config-file $(PYPIRC_PATH) $(Pipeline.Workspace)/${{parameters.ArtifactName}}/$(Package.Name)/*a*.whl
```

Comment:
Not really what this change is about, but I wonder if this wildcard is too permissive.

Reply:
What makes you say that?